### ISSUE TITLE
Option to disallow scoped

### ISSUE THREAD

**Issue by racoleman** [future-architect/eslint-plugin-vue-scoped-css#29]
Thanks for making this plugin! Our team has a slightly different use case: we'd like to enforce _no_ `scoped` attribute, as we find the side effects of scoped CSS to be too risky. Any chance of modifying the `require-scoped` rule to allow for "always"/"never" options?

**Comment by shadow7412** [comment]
Further to this suggestion - we have one spot in the application (App.vue in our case) where we have an intentionally unscoped style tag.

How do you suppress this rule for one specific tag?

**Comment by ota-meshi** [comment]
Hi @racoleman! Thank you for posting this issue.
I think it is good to add an option. The name of the rule "require-scoped" looks strange, but it should be fine.

**Comment by ota-meshi** [comment]
Hi @shadow7412. The easiest way is to use configuration comments.

https://future-architect.github.io/eslint-plugin-vue-scoped-css/playground/#eJx9jEEKgCAQAL8ie4xE6CjiS7zpCoKouRqE+PdAqGO3YWBmgM0OQYIiW0Np2iSxMaQYUmNXR042F3TcEomKZw/1VZJl79kmTFLiaxW1O6JebhHsUHtEAjngdwfymPMB/lEykw==

```vue
<script>
/* eslint vue-scoped-css/require-scoped: off */
</script>
<style>
</style>
```

Depending on the structure of your project, you can also use the cascading or `overrides` option.

https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy
https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns

**Comment by shadow7412** [comment]
Thanks. I ended up adding an override in .eslintrc - which also did the
trick.
